### PR TITLE
fix: various QOL, other fixes

### PR DESCRIPTION
**Pull request by ryanzhoudev**
- /lootrun save <name> now checks if a lootrun with that name already exists
- Various grammar/docs changes
- /wynntils version now copies version information to clipboard with format `Wynntils Version 1.11.3 Build 1519`
- Party finder queues now properly updates party information when queuing by yourself

### CODE REVIEW

File: src/main/java/com/wynntils/modules/map/commands/CommandLootRun.java [hunk 1]
```diff
@@ -98,6 +98,9 @@ public void execute(MinecraftServer server, ICommandSender sender, String[] args
                     throw new WrongUsageException("/lootrun save [name]");
                 }
                 String name = args[1];
+                if (LootRunManager.hasLootrun(name)) {
+                    throw new CommandException("A loot run with this name already exists!");
```
Comment (kristofbolyai) [inline src/main/java/com/wynntils/modules/map/commands/CommandLootRun.java]:
```suggestion
                    throw new CommandException("A lootrun file with this name already exists!");
```

**Comment by kristofbolyai** [comment]
Otherwise LGTM

File: src/main/java/com/wynntils/modules/map/commands/CommandLootRun.java (hunk 1 above)
Comment (ryanzhoudev) [reply src/main/java/com/wynntils/modules/map/commands/CommandLootRun.java]: Want me to change all instances of that in the file?

Comment (kristofbolyai) [reply src/main/java/com/wynntils/modules/map/commands/CommandLootRun.java]: Yeah

**Review by magicus — APPROVED** [review]

**Review by Incompleteusern — APPROVED** [review]

**Pull request merged**